Topics/Tech_Stacks/PostgreSQL_psycopg2.md

```diff
@@ -0,0 +1,74 @@
+# Learning PostgreSQL and psycopg2
+
+### Prerequisites
```

Comment:
The pre-requisites section is helpful for setting expectations. It might be good to include sub-points about what specifically the user should know. For example, in the “Basic operations” section you talk about the “select” operation, but is the user's knowledge of how to import a table pre-requisite knowledge?

Reply:
Made more specific pre-requisite section in md file.